machinist/mnode/assets/install_libpam-script.sh

```diff
@@ -0,0 +1,14 @@
+declare -A osInfo;
```

Comment:
arrays are not POSIX shell features, only a subset of shells support arrays.
Start the script with:

```
#!/usr/bin/env bash
```

or even `/bin/bash`, most systems have bash installed today.

Reply:
Done